api/Controllers/WarehouseController.cs

```diff
@@ -21,13 +21,33 @@ public ActionResult<Warehouse> GetById(Guid id)
         return Ok(foundWarehouse);
     }
 
-    [HttpGet("all")]
-    public ActionResult<IEnumerable<Warehouse>> GetAll()
+    [HttpGet()]
+    public ActionResult<IEnumerable<Warehouse>> GetAll() => Ok(_warehouseProvider.GetAll().Select(ig => new WarehouseResponse
     {
```

Comment:
Overal waar je "ig" hebt gezet verander naar wh of iets, ig staat voor Item Group...